### PR TITLE
Update GIT_AND_GITHUB.md

### PR DESCRIPTION
**Pull request by ealmdahl**
Add part about doing physical, pair-peer-reviews

### CODE REVIEW

**Comment by fespinoza** [comment]
:+1:

File: GIT_AND_GITHUB.md [hunk 1]
```diff
@@ -57,6 +57,21 @@ post the sufficient amount of gifs.
 
 Once the pull request is accepted, the person who merged should delete the branch.
 
+#### Reviewing pull requests
+
+As a reviewer, you should ideally be a core team member and have enough context
+to make a thorough review of the committed code, just by reading it. However, 
+there are times when the pull request is a bit on the large side, or it is 
+referencing existing code not found in the PR, or you for some other reason do 
+not have a good enough context to review enything else than code style and typos.
```
Comment (zenangst) [inline GIT_AND_GITHUB.md]: Typo: enything -> anything

**Comment by zenangst** [comment]
I approve of this! 👍👍👍

File: GIT_AND_GITHUB.md (hunk 1 above)
Comment (ealmdahl) [inline GIT_AND_GITHUB.md]: JESUS CHRIST! HOW DID THAT HAPPEN!?

(thanks, fixing meow :cat: )

**Comment by attila-at-hyper** [comment]
This reminds me that all the projects I'm currently working on (Dansani Plus, REMA Timeaksept, REMA Sekundering) are all "one man show" projects (from development point of view, of course), which makes detailed code reviews somewhat difficult.

**Comment by timkurvers** [comment]
Great stuff! Might also be a nice spot to remind people that any relevant discussions, oddities or trains of thought between the participants should make their way into either commit messages, code comments or documentation.

**Comment by zenangst** [comment]
@attila-at-hyper we need to break that bad cycle that we are in right now. One should never be alone on a project :)

File: GIT_AND_GITHUB.md [hunk 2]
```diff
@@ -57,6 +57,21 @@ post the sufficient amount of gifs.
...
+there are times when the pull request is a bit on the large side, or it is 
+referencing existing code not found in the PR, or you for some other reason do 
+not have a good enough context to review anything else than code style and typos. 
+In these cases, we encourage you to physically (or virtually) approach the PR 
+submitter and go through the pull request together.
+
+Be warned, though: By doing this, you will both need to explain your thoughts and
+discuss other alternatives. Of course, this means you are putting yourself at risk 
+of sharing your knowledge and/or learning some new stuff, so please be careful not
+to end up being even more awesome than you are.
```
Comment (jgorset) [inline GIT_AND_GITHUB.md]: Ha ha.

**Comment by jgorset** [comment]
Cool! It seems that we are all agreed, so I'ma merge this.

**Pull request merged**